AnywhereExamplePanel/js/anywhereHTTPApi.js

```diff
@@ -0,0 +1,106 @@
+/*  
+ * ADOBE SYSTEMS INCORPORATED
+ * Copyright 2015 Adobe Systems Incorporated
+ * All Rights Reserved.
+ * 
+ * NOTICE:  Adobe permits you to use, modify, and distribute this file in accordance with the 
+ * terms of the Adobe license agreement accompanying it.  If you have received this file from a 
+ * source other than Adobe, then your use, modification, or distribution of it requires the prior 
+ * written permission of Adobe.
+ */
+
+/***************************************************************************************
+ * -- AnywhereHTTPApi ------------------------------------------------------------------------
+ * 
+ * This wraps http API calls to the Anywhere REST API using simple JQuery $.ajax calls
+ * 
+ **************************************************************************************/
+var AnywhereHTTPApi = (function(exports) {
+    
+    /**
+    * This is a helper to step through the discoverable API. All entitiy pages have a 
+    * link section with functionality or further resources to discover.
+    * This functions returns the 'href' of a link basen on the 'rel' string.
```

Comment:
This functions returns => This function returns
of a link basen => of a link based